### ISSUE TITLE
macOS Testers Wanted

### ISSUE THREAD

**Issue by complexlogic** [complexlogic/loudgain#102]
Static binaries will be provided for macOS in the next release. I can verify that the builds are successful via the GitHub Actions CI, however, I have no means to test the builds because I don't have access to a Mac computer. Could any Mac owners test out the either one of the builds below and report if you experience any issues?
- [x86-64 build](https://github.com/complexlogic/rsgain/actions/runs/7824237457/artifacts/1228973220)
- [ARM64 build](https://github.com/complexlogic/rsgain/actions/runs/7824237457/artifacts/1228973217)

**Comment by NaanProphet** [comment]
Could you confirm the minimum MacOS required? I'm guessing it's >= Monterey 12.7

**Comment by complexlogic** [comment]
Yes, I believe that is correct. Here is the [configuration](https://github.com/actions/runner-images/blob/0390d98adc9f16efe6c10bd48a8cd71eef9a7972/images/macos/macos-12-Readme.md) for the CI.

**Comment by NaanProphet** [comment]
So far I can report that the ARM64 build seems to work on the following!!

* 2023 16-inch M2 MacBook Pro (Mac14,10)
* MacOS 14.2.1 Sonoma

Sample output:

```
% ./rsgain custom --album *
[✔] Scanning 'Track 01.m4a'
[✔] Container: QuickTime / MOV [mov,mp4,m4a,3gp,3g2,mj2]
[✔] Stream #0: ALAC (Apple Lossless Audio Codec), 16 bit, 44,100 Hz, 2 ch
 100% [=========================================================================]
[✔] Scanning 'Track 02.m4a'
[✔] Container: QuickTime / MOV [mov,mp4,m4a,3gp,3g2,mj2]
[✔] Stream #0: ALAC (Apple Lossless Audio Codec), 16 bit, 44,100 Hz, 2 ch
 100% [=========================================================================]
[✔] Scanning 'Track 03.m4a'
[✔] Container: QuickTime / MOV [mov,mp4,m4a,3gp,3g2,mj2]
[✔] Stream #0: ALAC (Apple Lossless Audio Codec), 16 bit, 44,100 Hz, 2 ch
 100% [=========================================================================]

Track: Track 01.m4a
  Loudness:   -25.93 LUFS
  Peak:     0.999390 (-0.01 dB)
  Gain:         7.93 dB 


Track: Track 02.m4a
  Loudness:   -22.61 LUFS
  Peak:     0.999390 (-0.01 dB)
  Gain:         4.61 dB 


Track: Track 03.m4a
  Loudness:   -28.35 LUFS
  Peak:     0.999969 (-0.00 dB)
  Gain:        10.35 dB 

Album:
  Loudness:   -24.85 LUFS
  Peak:     0.999969 (-0.00 dB)
  Gain:         6.85 dB 

%
```

I'll keep using it and trying it out! Please let me know if you'd like me to run any specific tests.

**Note on Gatekeeper**
I first received a `... can’t be opened because Apple cannot check it for malicious software.` message. To mark the exception, I first right clicked the binary in Finder, clicked Open, allowed it to run, which opened a new Terminal window and displayed the default help text and then completed. Then I could use custom commands in Terminal.

<img width="682" alt="image" src="https://github.com/complexlogic/rsgain/assets/2568736/b5a5ab08-7102-45f4-ad3b-b9218b7a6aa3">

**Comment by complexlogic** [comment]
Glad to see that it's working well. The Gatekeeper issue is likely because the binaries aren't digitally signed. I will add a note in the README about the necessary workaround.

Thanks a lot for your help.

**Comment by mjvotaw** [comment]
Testing it out on a 2019 Macbook Pro (2.6 GHz 6-Core Intel Core i7), MacOS Sonoma 14.3.1.
It seems to run fine, and gives the expected results (in that they're identical to the version installed through homebrew).

**Comment by complexlogic** [comment]
Builds are now available in [v3.5](https://github.com/complexlogic/rsgain/releases/tag/v3.5).
